packages/blocks/src/store/process-block-type.js

```diff
@@ -103,6 +103,19 @@ export const processBlockType =
 			),
 		};
 
+		// Temporary fix to allow block authors time to fix blocks that register attributes as null or undefined.
+		if (
+			! blockType.attributes ||
+			typeof blockType.attributes !== 'object'
+		) {
+			warning(
+				'The block "' +
+					name +
+					'" must not register attributes as `null` or `undefined`. Use an empty object (`attributes: {}`) or exclude the `attributes` key. In the next Gutenberg release, passing `null` or `undefined` for `attributes` will cause the block editor to crash.'
```

Comment:
Also, this "in the next release it will crash", isn't accurate, is it? I think we can omit that whole sentence.